### PR TITLE
using typescript in a smart way across all projects

### PR DESCRIPTION
**Pull request by ste00martin**
basically, allowing us to use typescript in whichever environment we wanna without extending `enviroment/node/typescript` just plain ol `environment/node` or `environment/react` etc

### CODE REVIEW

File: environment/node/index.js [hunk 1]
```diff
@@ -5,4 +5,5 @@ module.exports = { // eslint-disable-line no-undef
   extends: '../../index.js',
   globals: { Promise: true },
   parser: 'babel-eslint',
```
Comment (ryanliszewski) [inline environment/node/index.js]: Will this parser override the `typescript-eslint-parser`?

Comment (ste00martin) [reply environment/node/index.js]: it wont but you're right it doesn't need to be here any longer; removed

File: README.md [hunk 2]
```diff
@@ -33,22 +31,35 @@ Next get precommit hooks setup by adding the following to `package.json`:
 }
 ```
 
-## Step 4: Add in Visual Studio Code settings to repo
-TODO: automate this by copying this on package install
+# Testing New Linting Rules
+How do I know that the new eslint-config-volta will work good? Glad you asked partner! With our handy dandy `yarn link` testing's a doozy. Let me show yall:
```
Comment (ryanliszewski) [inline README.md]: Partner, ey?

**Comment by ryanliszewski** [comment]
I'll let you do the honors in publishing this!

**Pull request merged**